cnd/src/db/new_types.rs

```diff
@@ -0,0 +1,44 @@
+use ethereum_support::{FromDecimalStr, U256};
```

Comment:
:upside_down_face: You could rename the file to `newtype.rs` since I think it's [the correct term](https://wiki.haskell.org/Newtype) :stuck_out_tongue:

Reply:
:question: Also, would it make sense for this one to be something like `ledger_newtypes.rs` and the other one to go from `custom_sql_types.rs` to `sql_newtypes.rs`? It may make the separation clearer.

Reply:
I like the idea!

Would you mind sending a follow-up PR to get it in? :)